run/build

```diff
@@ -16,6 +16,8 @@ checkReturnCode()
 	fi
 }
 
+export NODE_ENV=${NODE_ENV:-"production"}
```

Comment:
Should be some condition. I don't think it's a good idea to just leave production env for every `run/built` case

Reply:
You are right in one thing - i need to change "production" to "develop". It was set to "production" to check if uglifying works on the production
In other thing:
it works like that:
 take a variable NODE_ENV, it there is not, set to mode, which is mentioned after ":".
 In that case it is production.
The value for NODE_ENV on the production is taken from kubefiles - configuration.yaml -  NODE_ENV: production.